components/eam/src/physics/cam/shoc.F90

```diff
@@ -31,6 +32,9 @@ module shoc
 real(rtype), parameter, public :: largeneg = -99999999.99_rtype
 real(rtype), parameter, public :: pi = 3.14159265358979323_rtype
 
+!repeated from shoc_intr!
+real(rtype), parameter         :: p0_shoc = 100000._rtype
```

Comment:
We should probably pass this through `shoc_init` as well to make sure everything matches. It is treated as an EAMxx constant so that way no sneaky BFB issues pop up. 

This requires some EAMxx/EAM interface changes, let me know if you want and I can add a quick commit.